pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "noisepy-seis"
+dynamic = ["version"]
+description = "A High-performance Computing Python Package for Ambient Noise Analysis"
+readme = "README.md"
+license = {file= "LICENSE"}
+requires-python = ">=3.8,<=3.10"
+authors = [
+    { email = "mdenolle@fas.harvard.edu", name = "Marine Denolle" },
+    { email = "chengxin_jiang@fas.harvard.edu", name = "Chengxin Jiang" },
+]
+keywords = [
+    "ambient",
+    "change",
+    "cross-correlation",
+    "dispersion",
+    "monitoring",
+    "noise",
+    "seismic",
+    "surface",
+    "velocity",
+    "wave",
+]
+classifiers = [
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+]
+dependencies = [
+    "h5py>=3.8.0",
+    "mpi4py>=3.1.4",
+    "numba>=0.56.4",
+    "numpy>=1.22.0",
+    "pandas>=1.5.3",
+    "pyasdf>=0.7.5",
+    "pycwt>=0.3.0a22",
+    "wheel>=0.38.4",
+]
+
+
+[project.urls]
+Homepage = "https://github.com/mdenolle/NoisePy"
+
+[tool.hatch.version]
+path = "src/noisepy/seis/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "src/noisepy",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/noisepy"]
+
+[project.optional-dependencies]
+dev = [
+    "pytest>=7.2.2",
+    "memory-profiler>=0.61",
+    "pre-commit>=3.2",
+    "hatchling>=1.14"
```

Comment:
This is already required by the build system, shouldn't need to list this out for dev